docs/SQLModelingStandard.md

```diff
@@ -62,6 +62,8 @@ infrastructure will be re-designed to follow this guideline.*
   primary key should be `MyTableID`).
   - *i.e. the name of the primary identifier of the `country` table should be 
   `CountryID`.*
+> If a relational table requires a primary key ID (instead of the usual composite  
```

Comment:
I don't see the need for this.  <TableNameID> should be enough. 
This modification should be in a separate PR.
It should also be below the next point if you want to complement it. You now have 2 contradicting points (Line 65 and 67)

Reply:
Without this statement, the name of my primary key does not respect this document. it should be `ProjectSubprojectRelID` but I have `ProjectSubprojectTupleID`

And I dont see the contradiction with line 67, this line says that rel tables specifically should name their keys as such. line 67 says that most tables should have an autoincrement primary key

Reply:
Sorry, contradict lines 59-64.

Does change my comment about should be in a separate PR. If we create exceptions in the standard every time someone doesn't like the standard, why have standards in the first place?

Reply:
I see this less as a contradiction and more as a specification, line 59 applies to all tables EXCEPT rel tables.

And not sure I would say this is trying to bypass the standards, I see this as amending the standards to include a usecase we had not thought of before... so more like complementing it and standards are not set in stone they should be updated if need be.

As far as the suggestion itself initially made by @driusan I kind of agree with the fact that `ProjectSubprojectTupleID` is clearer than `ProjectSubprojectID` or `ProjectSubprojectRelID` so the amendment here would be actually improving our standards.

What do you think ??

Reply:
I don't see any 'EXCEPT' in the standard, so 'ProjectSubprojectRelID' is in line with the current standard.  If you want to modify/complement/improve the standard, it should be on a separate PR where people can study and comment appropriately.

Reply:
So in this PR I add a new column called `ProjectSubprojectRelID` wait for that to get merged and then open another one changing it to `tuple` ?

Reply:
create a PR to modify the standard, once approved (if approved as nobody can predict the future, I surely can't) then use whatever the standard say.

Reply:
ORRRRR just call it rel

I pick option number 2